specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2025-01-01/examples/ReplicationProtectionClusters_Create.json

```diff
@@ -0,0 +1,198 @@
+          "healthErrors": [],
+          "lastSuccessfulFailoverTime": null,
+          "policyId": "/Subscriptions/c183865e-6077-46f2-a3b1-deb0f4f4650a/resourceGroups/resourceGroupPS1/providers/Microsoft.RecoveryServices/vaults/vault1/replicationPolicies/24-hour-retention-policy",
+          "policyFriendlyName": "24-hour-retention-policy",
+          "currentScenario": null,
+          "recoveryContainerId": "/Subscriptions/c183865e-6077-46f2-a3b1-deb0f4f4650a/resourceGroups/resourceGroupPS1/providers/Microsoft.RecoveryServices/vaults/vault1/replicationFabrics/centraluseuap/replicationProtectionContainers/centraluseuap-container",
+          "agentClusterId": null,
+          "clusterFqdn": null,
+          "clusterNodeFqdns": [],
+          "clusterProtectedItemIds": [],
+          "provisioningState": "Created",
+          "areAllClusterNodesRegistered": false,
+          "clusterRegisteredNodes": [],
+          "providerSpecificDetails": {
+            "instanceType": "A2A",
+            "multiVmGroupId": null,
+            "multiVmGroupName": null,
+            "multiVmGroupCreateOption": null,
+            "primaryFabricLocation": "eastus",
+            "recoveryFabricLocation": "centraluseuap",
+            "failoverRecoveryPointId": null,
+            "clusterManagementId": "4b8c1c01-5ad7-4958-a98d-daf7cca6a124",
+            "rpoInSeconds": null,
+            "lastRpoCalculatedTime": null,
+            "initialPrimaryZone": "",
+            "initialPrimaryFabricLocation": "eastus",
+            "initialRecoveryZone": "",
+            "initialRecoveryFabricLocation": "centraluseuap",
+            "initialPrimaryExtendedLocation": null,
+            "initialRecoveryExtendedLocation": null,
+            "primaryAvailabilityZone": null,
+            "recoveryAvailabilityZone": null,
+            "primaryExtendedLocation": null,
+            "recoveryExtendedLocation": null,
+            "lifecycleId": "d5f6f011-7ef5-4c78-80fe-6e1b28401c4d"
+          },
+          "sharedDiskProperties": {
+            "protectionState": "EnablingProtection",
+            "activeLocation": null,
+            "allowedOperations": [],
+            "replicationHealth": "Normal",
+            "healthErrors": [],
+            "currentScenario": null,
+            "sharedDiskProviderSpecificDetails": {
+              "instanceType": "A2A",
+              "managementId": "4b8c1c01-5ad7-4958-a98d-daf7cca6a124",
+              "unprotectedDisks": null,
+              "protectedManagedDisks": [],
+              "primaryFabricLocation": "eastus",
+              "recoveryFabricLocation": "centraluseuap",
+              "failoverRecoveryPointId": null,
+              "monitoringPercentageCompletion": null,
+              "monitoringJobType": null,
+              "rpoInSeconds": null,
+              "lastRpoCalculatedTime": null,
+              "sharedDiskIRErrors": null
+            }
+          }
+        }
+      }
+    },
+    "201": {
```

Comment:
>201 [](http://example.com/codeflow?start=5&length=3)

As discussed pls remove this from example as well.

Reply:
Updated